scripts/devicefarm-test-runner-buildspec.yml

```diff
@@ -61,4 +63,11 @@ artifacts:
   files:
     - '**/*.apk'
   name: AmplifyAndroidCatApks
-  discard-paths: yes
\ No newline at end of file
+  discard-paths: yes
+  secondary-artifacts:
+    devicefarmRunReport:
+      files:
+        - '**/*'
+      name: DevicefarmTestRunReport
+      base-directory: 'build/reports/dfrun'
```

Comment:
I figured out the `DF` acronym from context, but there might be a better name here `build/reports/instrumented` or something? (And for line 51, to match.)

Reply:
Updated to instrumented